lib/restaurant.rb

```diff
@@ -0,0 +1,15 @@
+class Restaurant
+
+  attr_reader :menu
+
+  def initialize(menu)
+    @menu = menu
+  end
+
+  def submit_order(customer_name, customer_phone_number, order_dishes)
+    order_dishes.each { |dish| fail "Dish not on menu" if !@menu.dishes.include? dish }
```

Comment:
Metrics/LineLength: Line is too long. [87/80]<br>Style/NegatedIf: Favor `unless` over `if` for negative conditions.